.env

```diff
@@ -47,3 +47,4 @@ ARROW_R_DEV=TRUE
 R_ORG=rhub
 R_IMAGE=ubuntu-gcc-release
 R_TAG=latest
+DEVTOOLSET_VERSION=-1
```

Comment:
This probably deserves a comment to say that the default (negative) means not to install a devtoolset